actors/builtin/power/power_state.go

```diff
@@ -15,10 +15,12 @@ import (
 )
 
 type State struct {
-	TotalRawBytePower     abi.StoragePower
-	TotalQualityAdjPower  abi.StoragePower
-	TotalPledgeCollateral abi.TokenAmount
-	MinerCount            int64
+	TotalRawBytePower         abi.StoragePower
+	TotalRawBytePowerNoMin    abi.StoragePower
```

Comment:
I don't love names, suggestions welcome

Reply:
Let's reserve "power" for the thing that grants the chance to mine a block. So, how about `TotalBytesCommitted`, `TotalQABytesCommitted`. Also please add a brief note here as to why they can differ.